kostyor/core/driver.py

```diff
@@ -0,0 +1,15 @@
+import json
+
+
+class Driver(object):
```

Comment:
I think that we should consider using #52 as the model - since steps may be internal to the implementation